recipes/default.rb

```diff
@@ -29,7 +32,7 @@
 
 directory node.loggly.tls.cert_path do
   owner 'root'
-  group 'syslog'
+  group node.rsyslog.priv_group || 'root'
```

Comment:
I can't remember if this was left from my refactor or needed for ubuntu.

Rather than use `||` would you put the default in the attributes.rb?